packages/scene-graph-mediator/client/example/UnityProject/.gitignore

```diff
@@ -0,0 +1,4 @@
+Library/
+Logs/
+Obj/
+Builds/
```

Comment:
Consider adding the following directories.
`Temp/` : Unity working cache directory.